Add studio to the readme

bilgeyucel opened the pull request:
Two versions:
* As a tip box (smaller)
* As a new section

TuanaCelik reviewed (commented):
I also prefer the section @bilgeyucel - if we can add the video that would be great. Also, how about "Visual Pipeline Editor" instead of "Visual Creator"

bilgeyucel commented:
> I also prefer the section @bilgeyucel - if we can add the video that would be great. Also, how about "Visual Pipeline Editor" instead of "Visual Creator"

Added the video and updated the title. You can take another look
